src/Microsoft.Health.Dicom.Functions.Client/Registration/DicomServerBuilderFunctionClientRegistrationExtensions.cs

```diff
@@ -54,7 +56,8 @@ public static class DicomServerBuilderFunctionClientRegistrationExtensions
             .GetSection(DicomFunctionOptions.SectionName)
             .GetSection(nameof(DicomFunctionOptions.DurableTask))
             .Bind(x));
-        services.Replace(ServiceDescriptor.Singleton<IMessageSerializerSettingsFactory, DurableTaskSerializerSettingsFactory>());
+        services.Configure<JsonSerializerSettings>(o => o.ConfigureDefaultDicomSettings());
+        services.Replace(ServiceDescriptor.Singleton<IMessageSerializerSettingsFactory, MessageSerializerSettingsFactory>());
```

Comment:
Here the `MessageSerializerSettingsFactory` type uses `IOptions<JsonSerializerSettings>` to control the serialization, which is configured right above. This way we an inject custom converters.